internal/controller/services/monitoring/monitoring_controller_support.go

```diff
@@ -124,6 +135,45 @@ func getTemplateData(ctx context.Context, rr *odhtypes.ReconciliationRequest) (m
 	return templateData, nil
 }
 
+func isSingleNodeCluster(ctx context.Context, rr *odhtypes.ReconciliationRequest) bool {
```

Comment:
any reason to go this way? isnt it enough to get the number of the nodes in the cluster to decide if it is SNO or not?

Reply:
realised I was overcomplicating it, I'll change it to just count the nodes